### PR TITLE
Sep deploy

### PR DESCRIPTION
**Pull request by CJ-Wright**
This makes a separate deploy functionality which pushes up to github. The main benefit is that we can reduce our time pushing things to github for cf-graph. Note that this has a sister PR in [`circle_worker`](https://github.com/regro/circle_worker/pull/2) both need to go in at the same time.

### CODE REVIEW

File: conda_forge_tick/cli.xsh [hunk 1]
```diff
@@ -8,7 +8,23 @@ from .make_graph import main as main1
 from .update_upstream_versions import main as main2
 from .auto_tick import main as main3
 
-int_script_dict = {0: main0, 1: main1, 2: main2, 3: main3}
+int_script_dict = {0: main0, 1: main1, 2: main2, 3: main3,
+                   -1: deploy}
```
Comment (scopatz) [inline conda_forge_tick/cli.xsh]: I think this dict needs to be defined below the definition of `deploy()` now

File: conda_forge_tick/cli.xsh [hunk 2]
```diff
@@ -8,7 +8,23 @@ from .make_graph import main as main1
...
+
+
+def deploy(script):
```
Comment (scopatz) [inline conda_forge_tick/cli.xsh]: Needs a docstring

**Comment by CJ-Wright** [comment]
@regro/auto-tick ready for more review/merge, note that this and https://github.com/regro/circle_worker/pull/2 need to be merged at the same time.

**Comment by CJ-Wright** [comment]
Thank you!

**Pull request merged**